src/notifications/endpoints/components/EndpointOverlayContents.tsx

```diff
@@ -41,6 +44,7 @@ const EndpointOverlayContents: FC<Props> = ({
     e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
   ) => {
     const {name, value} = e.target
+    event('Alert Panel (Notebooks) - Updated Endpoint')
```

Comment:
i think we just want this when selecting the type, not on every keystroke

Reply:
We already have that one, but you're right that would be awfully noisy.  Removed.